spec.html

```diff
@@ -4083,11 +4082,10 @@ <h1>Await Fulfilled Functions</h1>
 
         <emu-clause id="await-rejected">
           <h1>Await Rejected Functions</h1>
-
           <p>An Await rejected function is an anonymous built-in function that is used as part of the Await specification device to deliver the promise rejection reason to the caller as an abrupt throw completion. Each Await rejected function has an [[AsyncContext]] internal slot.</p>
-
           <p>When an Await rejected function is called with argument _reason_, the following steps are taken:</p>
 
+
```

Comment:
The ones which got changed here were an artifact of the process I was using. Whitespace is pretty inconsistent in general, so I figured it would be fine, but I can back those changes out if you'd prefer. I'd be happy to make it consistent in a separate PR, though I think would want to wait for the ability to enforce it (cf https://github.com/bterlson/ecmarkup/issues/173).

Reply:
I'm fine waiting on it; i'd just prefer to end up never having two blank lines in a row :-)

Reply:
Added to my [list](https://github.com/bterlson/ecmarkup/issues/173) of desiderata for linting the spec.